website/static/js/fangorn.js

```diff
@@ -2441,9 +2439,7 @@ function isInvalidDropItem(folder, item, cannotBeFolder, mustBeIntra) {
         // no dropping if waiting on waterbutler ajax
         item.inProgress ||
         (cannotBeFolder && item.data.kind === 'folder') ||
-        (mustBeIntra && item.data.provider !== folder.data.provider) ||
-        //Disallow moving OUT of figshare
-        item.data.provider === 'figshare'
```

Comment:
Is copying working? The logic looks like getCopyMode will return 'move' for figshare unless I missed something. 

`!(item.data.provider === 'figshare') ` should be added to `allowedToMove` to make getCopyMode return 'copy'